templates/components/carousel.html

```diff
@@ -62,8 +62,12 @@
     {{/each}}
     {{#and arrows (if carousel.slides.length '>' 1)}}
     <button aria-label="{{lang 'carousel.arrowAriaLabel'}} {{carousel.slides.length}}" class="js-hero-next-arrow slick-next slick-arrow"></button>
+    <span 
+        class="carousel-content-change-message aria-description--hidden" role="status">
```

Comment:
as this class used for logic but not for styles, can we add "js-" prefix to it as it is done for example for play/pause button?

Reply:
np